src/intervals/interval_operations/pointwise_boolean.jl

```diff
@@ -1,39 +1,32 @@
 # This file is part of the IntervalArithmetic.jl package; MIT licensed
 
-#=  This file contains functions not specified in the IEEE Std 1788-2015,
-    extending boolean operations defined for reals to intervals.
+#=  This file contains boolean operations defined for intervals.
 
-    This is different from what is described in sections 9.5 of the
-    IEEE Std 1788-2015 (Boolean functions of intervals), since we deal here
-    with what is needed for drop-in replacement of floats by intervals,
-    and not what is reasonnable to do with intervals in a separate execution
-    environment.
-
-    Essentially, julia is better at composability and we are more ambitious
-    that what the standard ever expected, so we have some extra problems
-    to solve.
+    As there is no common consensus on how these operations should be defined for intervals,
+    in addition to the behavior defined by the standard (sec. ), the file also implements
+    different policies to handle these operations, giving the user to choose the one most
+    suited for their use case.
```

Comment:
Benoit refers (systematically?) to the standard as "IEEE Std 1788-2015"; I think we should follow that. Is the section number you refer to, section 9.5? (Sorry for being too picky.)

I must admit that I really like the last paragraph you deleted: "Essentially, julia is better at composability and we are more ambitious that what the standard ever expected, so we have some extra problems to solve."

Reply:
> Benoit refers (systematically?) to the standard as "IEEE Std 1788-2015"; I think we should follow that. Is the section number you refer to, section 9.5? (Sorry for being too picky.)

hups sorry I left that hanging :)

> I must admit that I really like the last paragraph you deleted: "Essentially, julia is better at composability and we are more ambitious that what the standard ever expected, so we have some extra problems to solve."

Personally, I find the paragraph a little out of place here. Of course I think julia is cool and great and composability, but the right place for such a claim would be in a tutorial notebook, showing with some examples how one policy fails when combining with another package and another gives the desired results. API docs should stick to the raw facts and implementation details. I also fear it's a little misleading, the "we are more ambitious and have extra problems to solve" sounds to me like in the file new solutions to more advanced problems are proposed, while actually the other non-ieee policies are actually older and what other interval arithmetic libraries (filib, gaol, boost) already did before the standard.